### PR TITLE
workspace package metadata

### PR DESCRIPTION
**Pull request by gibbz00**
~~Depends on #1033~~ (merged into master and rebased here)

### CODE REVIEW

File: prost-build/Cargo.toml [hunk 1]
```diff
@@ -9,7 +9,6 @@ authors = [
 ]
 license = "Apache-2.0"
 repository = "https://github.com/tokio-rs/prost"
-documentation = "https://docs.rs/prost-build"
```
[the remark this answers is not in the document]
Comment (gibbz00) [reply prost-build/Cargo.toml:12]: https://doc.rust-lang.org/cargo/reference/manifest.html#the-documentation-field

> If no URL is specified in the manifest file, crates.io will automatically link your crate to the corresponding docs.rs page when the documentation...

File: prost-build/Cargo.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ authors.workspace = true
 license.workspace = true
 repository.workspace = true
 readme = "README.md"
-description = "A Protocol Buffers implementation for the Rust Language."
```
[the remark this answers is not in the document]
Comment (gibbz00) [reply prost-build/Cargo.toml:14]: Oki!

File: prost-build/Cargo.toml [hunk 3]
```diff
@@ -6,7 +6,7 @@ license.workspace = true
 repository.workspace = true
 readme = "README.md"
 description.workspace = true
-edition = "2021"
```
[the remark this answers is not in the document]
Comment (gibbz00) [reply prost-build/Cargo.toml:15]: Agreed!

[the remark this answers is not in the document]
Comment (gibbz00) [reply prost-build/Cargo.toml:15]: Separate PR: https://github.com/tokio-rs/prost/pull/1039

[the remark this answers is not in the document]
Comment (caspermeijn) [reply prost-build/Cargo.toml:15]: Now that the private projects have the same edition, then can also use the `workspace.edition`.

**Comment by caspermeijn** [comment]
> Now that the private projects have the same edition, then can also use the `workspace.edition`.

@gibbz00 Could you add `edition.workspace = true` to the all crates?

**Review by caspermeijn — APPROVED** [review]
Thanks for your contribution

**Pull request merged**